src/app_model/types/_keys/_key_codes.py

```diff
@@ -602,6 +658,22 @@ def _keycode_from_string(keystr: str) -> KeyCode:
         # sourcery skip
         return KEYCODE_FROM_LOWERCASE_STRING.get(str(keystr).lower(), KeyCode.UNKNOWN)
 
+    def _keycode_to_os_symbol(keycode: KeyCode, os: OperatingSystem) -> str:
+        """Return key symbol for an OS for a given KeyCode."""
+        if os == OperatingSystem.MACOS:
+            return MACOS_KEY_SYMBOLS.get(str(keycode), str(keycode))
+        elif os == OperatingSystem.LINUX:
+            return LINUX_KEY_SYMBOLS.get(str(keycode), str(keycode))
+        return WIN_KEY_SYMBOLS.get(str(keycode), str(keycode))
+
+    def _keycode_to_os_name(keycode: KeyCode, os: OperatingSystem) -> str:
+        """Return key name for an OS for a given KeyCode."""
+        if os == OperatingSystem.MACOS:
+            return MACOS_KEY_NAMES.get(str(keycode), str(keycode))
+        elif os == OperatingSystem.LINUX:
+            return LINUX_KEY_NAMES.get(str(keycode), str(keycode))
+        return WIN_KEY_NAMES.get(str(keycode), str(keycode))
```

Comment:
```suggestion
    def _keycode_to_os_symbol(keycode: KeyCode, os: OperatingSystem) -> str:
        """Return key symbol for an OS for a given KeyCode."""
        if keycode in (symbols := OS_KEY_SYMBOLS.get(os, {})):
            return symbols[keycode]
        return str(keycode)

    def _keycode_to_os_name(keycode: KeyCode, os: OperatingSystem) -> str:
        """Return key name for an OS for a given KeyCode."""
        if keycode in (names := OS_KEY_NAMES.get(os, {})):
            return names[keycode]
        return str(keycode)
```

note, this also avoids the additional lookup triggered by `str(keycode)` until absolutely necessary